src/common/cockpitmemfdread.c

```diff
@@ -62,6 +62,13 @@ cockpit_memfd_read (int      fd,
       return NULL;
     }
 
+  if (buf.st_size > 100000)
```

Comment:
I have a (mild) aversion against magic numbers in the middle of code. Would you consider moving it to a `#define MAX_SIZE` at the top? Or turn it into a function argument, so that the caller can decide which maximum size is reasonable?